### ISSUE TITLE
Issue with Numpy 1.13.0 compatibility

### ISSUE THREAD

**Issue by MeganNewsome** [LSST-nonproject/sims_maf_contrib#48]
At line 193, lcNumber is defined in which np.floor produces a floating value. This becomes in an issue in the later for loop as it needs to be a solid integer. Small fix once found, but not at all intuitive that it would be the problem, as it works up to numpy version 1.12.2. but not with numpy 1.13.

`lcNumber = np.floor((dataSlice[self.mjdCol] - surveyStart) / self.transDuration) + lcNumberStart`

This was fixed by converting the floating value to integer in the following line:

`lcNumber = np.array(lcNumber, dtype=np.int)`

So an updated code for users who intend on using the latest version of sims_maf should have the following at lines 193/194:

`lcNumber = np.floor((dataSlice[self.mjdCol] - surveyStart) / self.transDuration) + lcNumberStart
 lcNumber = np.array(lcNumber, dtype=np.int)`

**Comment by MeganNewsome** [comment]
Note this is with reference to "transientAsciiMetric.py".

**Comment by rhiannonlynne** [comment]
This is definitely a fix to put in, but I'm actually a little confused -- the stack itself won't build with numpy 1.13 yet (hopefully next week .. there's a ticket in progress) yet.

**Comment by MeganNewsome** [comment]
Odd, it says that my version of numpy is 1.13 from the Miniconda python running in MAF. This metric wouldn't run on my shell, whereas it would run on the shell of my colleague's. After running "source loadLSST.bash" and "setup sims_maf -t sims" like normal to ensure the python being used is the Miniconda as established within MAF, we then found that our versions of numpy were different. Regarding installation, we each followed the exact same installation instructions, **except** that I had ran an eups update on mine to ensure the most recent sims_maf on my system, while she installed hers without updating it afterwards. So we assumed the most recent version of MAF must use numpy 1.13 since that's what it's running in mine?

**Comment by rhiannonlynne** [comment]
Eups update or conda update? (There is no eups update command). A conda
update would update the conda packages, including numpy, to the most recent
versions, ahead of those which come standard with the lsst miniconda.

**Comment by MeganNewsome** [comment]
I ran this line:
`eups distrib install sims_maf -t sims      # this will install the latest version (specific version tags not working)`

from this page: https://confluence.lsstcorp.org/display/SIM/Install+or+Update+MAF

to update my packages upon first installation. I didn't have my colleague do the same. I don't recall nor have it written down that I performed a conda update but I will try to check again.
